extensions/amp-date-picker/amp-date-picker.md

```diff
@@ -334,12 +352,62 @@ By default, this attribute is not present.
 
 ##### src [optional]
 
-If present, `amp-date-picker` requests JSON data to populate the
-`highlighted` and `blocked` lists, as well as matching templates in the document
-to lists of dates.
+If present, `amp-date-picker` requests JSON data to populate certain attributes
+dynamically, as well as matching lists of dates to template `id`s for
+rendering days in the calendar.
+
+###### blocked
```

Comment:
What's the difference between this attribute and the optional one (on line 335)?

Reply:
This needs to be clarified that this attribute and others are specific to using `src` attribute (if that's the case).

Reply:
Yes, this is a property of the response object from the URL specified by the `src` attribute. I'll add some clarification